soc/nxp/imx/imx9/imx943/Kconfig.defconfig.mimx943.a55

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMX94398_A55
+
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_FLASH := zephyr,flash
+
+config FLASH_SIZE
+	default $(dt_chosen_reg_size_int,$(DT_CHOSEN_Z_FLASH),0,K)
+
+config FLASH_BASE_ADDRESS
+	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_FLASH))
+
+# Enable GIC Safe Configuration to run multiple OSes on Cortex-A Cores
+config GIC_SAFE_CONFIG
+	default y
+
+config NUM_IRQS
+	default 320
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 24000000
```

Comment:
reminder (non blocking) that this needs to come from dts and PRs will be nacked in a weeks time if they do not

Reply:
Could you please provide an example? I find most of the item is defined in Kconfig and can't find how it is defined by dts.

Reply:
e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/gd/gd32/gd32e50x/Kconfig.defconfig.gd32e507#L7 but does not need fixing in this PR